assets/js/atomic/blocks/product/title/attributes.js

```diff
@@ -9,4 +14,24 @@ export const blockAttributes = {
 	},
 };
 
+if ( isFeaturePluginBuild() ) {
+	blockAttributes = {
+		...blockAttributes,
+		align: {
+			type: 'string',
```

Comment:
For align, I choose not to set an align, I left the default value to be set in CSS, so that themes and RTL can reset it easily, having it here won't make it RTL compatible.

For the rest, setting a default value will mess with the theme styles.

Reply:
Shouldn't it default left, but our RTL stylesheet substitute left for right? That's how it usually works. is-rtl is at body level.

Reply:
yes, when doing this via CSS, this is how it will work, but if we supply a default value in attributes, that value will be used, giving the block that class, and causing it to be left always (I RTL-ignored those classes to avoid conflict).